prerequisiteversions

```diff
@@ -0,0 +1 @@
+CMake=3.7.2
```

Comment:
This file is not required.  Other one that begins with a dot as in ".prerequisiteversions" is required.